### ISSUE TITLE
Full query only on one server

### ISSUE THREAD

**Issue by Bleala** [ddworken/hishtory#131]
Hello!

I discovered Hishtory this week and wanted to try it, because the features look very nice. :)

But I have one issue, the full `hishtory query` output is only displayed on one of my servers. On the other servers I only can see the output of this server and only of one user.

I installed it on three servers for my normal user and the root user, like it is described in the `README.md`.

1. `curl https://hishtory.dev/install.py | python3 -`
2. (For every server/user after the first one) `hishtory init MY_SECRET`

On the first server I installed everything, I can see the output from all other servers and the two user from each server.
On the other two servers I only can see the output from the specific server and user.

Do you have an idea why?

I'm using the Linuxserver.io docker image with Postgres as backend, if this is important.

Greetings

**Comment by GRbit** [comment]
@ddworken does it look like the issue could be closed?
TLDR: There was a problem with personal self-hosted server, but with your hishtory.dev it worked flawlessly.

Note from my side: I was able to make it work with self-hosted instance without much trouble. Most probably it was some local web server setting issue, rather than a hishtory bug.
